framework/doc/content/source/functions/PiecewiseConstantFromCSV.md

```diff
@@ -0,0 +1,42 @@
+# PiecewiseConstantFromCSV
+
+!syntax description /Functions/PiecewiseConstantFromCSV
+
+## Description
+
+The `PiecewiseConstantFromCSV` function is used to load data from a CSV file into a function. The
+[PropertyReadFile.md] user object takes care of reading the CSV file, and the function queries
+information from it.
+
+It can assume CSV data
+
+- is sorted by element-id, in which case, when the function is evaluated at a point, it will locate the element containing it then return the value for that element in the CSV file
```

Comment:
should the first column be the element ID/ node ID? these ID can be non contiguous

Reply:
to clarify, the active elements may have non contiguous IDs

Reply:
Ok I added the capability to deal with non-contiguous ID since I think the restriction was making this whole object rather limited in capability.
This took a little more development though, which you will find in the latest commit.

Reply:
Removed that commit. Just clear docs